Example/Tests/UserTests.swift

```diff
@@ -33,13 +33,12 @@ class UserTests: QuickSpec {
                 expect(newTokens?.refreshToken).toNot(equal(oldTokens?.refreshToken))
             }
 
-            it("Should emit set event when session refreshes it") {
+            it("Should update tokens when session refreshes it") {
                 self.stub(uri("/oauth/token"), try! Builders.load(file: "valid-refresh", status: 200))
                 let user = TestingUser(state: .loggedIn)
                 let oldTokens = user.wrapped.tokens
-                var newTokens: TokenData?
-                _ = user.wrapped.didSetTokens.register { newTokens = $0.new }
                 user.refresh { _ in }
+                let newTokens = UserTokensKeychain().data().first
```

Comment:
Should this be `let newTokens = user.wrapped.tokens` ?

Or maybe by "update tokens" in the test name it's meant that it should be updated in the keychain?